app_dart/lib/src/service/scheduler.dart

```diff
@@ -1085,64 +1137,37 @@ $stackTrace
     await unlockMergeQueueGuard(slug, sha, checkRunGuard);
   }
 
-  Future<void> _proceedToCiTestingStage({
-    required cocoon_checks.CheckRun checkRun,
-    required RepositorySlug slug,
-    required String sha,
-    required String mergeQueueGuard,
+  Future<void> _runCiTestingStage({
+    required PullRequest pullRequest,
+    required String checkRunGuard,
     required String logCrumb,
+    bool skipEngine = false,
+    String? flutterPrebuiltEngineVersion,
   }) async {
-    final checkRunGuard = checkRunFromString(mergeQueueGuard);
-
-    // Look up the PR in our cache first. This reduces github quota and requires less calls.
-    PullRequest? pullRequest;
-    final id = checkRun.id!;
-    final name = checkRun.name!;
-    try {
-      pullRequest = await findPullRequestFor(
-        firestoreService,
-        id,
-        name,
-      );
-    } catch (e, s) {
-      log.warning('$logCrumb: unable to find PR in PrCheckRuns', e, s);
-    }
-
-    // We'va failed to find the pull request; try a reverse look it from the check suite.
-    if (pullRequest == null) {
-      final int checkSuiteId = checkRun.checkSuite!.id!;
-      pullRequest = await githubChecksService.findMatchingPullRequest(slug, sha, checkSuiteId);
-    }
-
-    // We cannot make any forward progress. Abandon all hope, Check runs who enter here.
-    if (pullRequest == null) {
-      throw 'No PR found matching this check_run($id, $name)';
-    }
-
     try {
       // Both the author and label should be checked to make sure that no one is
       // attempting to get a pull request without check through.
       if (pullRequest.user!.login == config.autosubmitBot &&
           pullRequest.labels!.any((element) => element.name == Config.revertOfLabel)) {
-        log.info('$logCrumb: skipping generating the full set of checks for revert request.');
+        log.info('$logCrumb: skipping generating the full set of changes.');
```

Comment:
This is not skipping "changes" though. It's skipping checks.

Reply:
This was a mistake on the part of John and I, we paired and vscode made some weird edits.

I'll send a PR fixing this.